### PR TITLE
Feature/5

### PR DESCRIPTION
**Pull request by acadapp**
Producten page added

### CODE REVIEW

File: Models/Product.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace badaboom.Models
+{
+    public enum Currency
+    {
+        USD = 840,
```
Comment (vicmosin) [inline Models/Product.cs]: @acadapp 
why values are so strange? Can't we just use "normal" order, i.e. 0, 1, 2 etc.. ?

File: Views/Home/Index.cshtml [hunk 2]
```diff
@@ -10,7 +10,11 @@
         </ul>
     </div>
     <div class="col-md-6">
+
         PRODUCTS WILL BE HERE !!
```
Comment (vicmosin) [inline Views/Home/Index.cshtml]: Please add the list of products at home view as well, not just a link

File: Models/Product.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        EUR = 978,
+        ILS = 376,
+        RUR = 810
+    }
+
+    public class Product
```
Comment (vicmosin) [inline Models/Product.cs]: As agreed, we inherit from AbstractModel which has already Id field

File: Models/Product.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        public int Id { get; set; }
+        public Guid gId { get; set; }
```
Comment (vicmosin) [inline Models/Product.cs]: Remove that field, why do we need 2 different identifiers?

File: Views/Home/Products.cshtml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
```
Comment (vicmosin) [inline Views/Home/Products.cshtml]: Copy-paste is evil!

File: badaboom.sln [hunk 6]
```diff
@@ -0,0 +1,22 @@
+﻿
```
Comment (vicmosin) [inline badaboom.sln]: Please remove that file from PR

File: badaboom.xproj [hunk 7]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (vicmosin) [inline badaboom.xproj]: Please remove this file from PR

File: Models/Product.cs (hunk 1 above)
Comment (acadapp) [inline Models/Product.cs]: There are international currency codes, ISO 4217. For more information visit this page http://www.nationsonline.org/oneworld/currencies.htm

Comment (vicmosin) [inline Models/Product.cs]: And what exactly benefit of this except the fact we are ISO-compatible? ))

File: Views/Home/Products.cshtml (hunk 5 above)
Comment (acadapp) [inline Views/Home/Products.cshtml]: Removed, but it not copy-paste, it's autogenerated

File: Models/Product.cs (hunk 3 above)
Comment (acadapp) [inline Models/Product.cs]: Done

File: Models/Product.cs (hunk 4 above)
Comment (acadapp) [inline Models/Product.cs]: Done

**Comment by vicmosin** [comment]
@acadapp as I wrote, please merge master's changes first..

**Pull request merged**